lib/proto/ark/wallet_pb.js

```
@@ -0,0 +1,2495 @@
+ * @param {!jspb.BinaryReader} reader The BinaryReader to use.
+ * @return {!proto.fulmine.v1.GenSeedResponse}
+ */
+proto.fulmine.v1.GenSeedResponse.deserializeBinaryFromReader = function(msg, reader) {
+  while (reader.nextField()) {
+    if (reader.isEndGroup()) {
+      break;
+    }
+    var field = reader.getFieldNumber();
+    switch (field) {
+    case 1:
+      var value = /** @type {string} */ (reader.readString());
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)